contrib/mhchem/mhchem.js

```diff
@@ -0,0 +1,1687 @@
+﻿/* eslint-disable */
```

Comment:
Could you include the diff (patch) file between MathJax mhchem and this?

Reply:
@ylemkimon Where would you like to see the diff file? In the `contrib` folder? Here in this string of comments?